server/store/ip_store.go

```diff
@@ -75,10 +75,10 @@ func RegisterIPStoreDriver(name string, driver IPStoreDriver) {
 }
 
 // OpenIPStore returns an IPStore specified by a configuration.
-func OpenIPStore(cfg *DriverConfig) (IPStore, error) {
+func OpenIPStore(cfg *DriverConfig) (IPStore, StopperFunc, error) {
```

Comment:
It does! But now we expose the (IP|String|Peer)Store.Stop() func to every user of the store :(